draft-ietf-mls-protocol.md

```diff
@@ -1623,6 +1624,9 @@ psk_[1] --> KDF.Extract --> ExpandWithLabel --> KDF.Extract = psk_secret_[1]
 psk_[n] --> KDF.Extract --> ExpandWithLabel --> KDF.Extract = psk_secret_[n]
 ~~~~~
 
+In particular, if there are no PreSharedKey proposals in a given Commit, then
+the resulting `psk_secret` is `psk_secret_[0]`, the all-zero vector.
```

Comment:
... of length `KDF.n`. (?)

Reply:
We specify the length just a couple of paragraphs above ("Here `0` represents..."), so I thought it was clear enough to leave it out here.